builtin/joined_dataset.cc

```diff
@@ -648,7 +648,7 @@ struct JoinedDataset::Itl
         auto it = columnIndex.find(columnName);
 
         if (it == columnIndex.end())
-            throw HttpReturnException(500, "Joined dataset did not find column with given hash",
+            throw HttpReturnException(400, "Joined dataset did not find column ",
```

Comment:
I wonder if this should not stay a 500.  `getColumn` is called in many contests where some might not have been triggered directly by a user input.  Can we catch that error later down the stack and convert it to a 400 when it is clear that the user input is the source of the problem?